butane/build.rs

```diff
@@ -0,0 +1,9 @@
+fn main() {
+    let dirs = [".butane/", "../example/.butane"];
```

Comment:
Could you add a comment in the build.rs giving some background for future readers on why the cleanup is happening?